lib/core/boot/boot.ts

```diff
@@ -71,32 +71,62 @@ export const bootstrap = (): FluxifyServer => {
 				try {
 					if (throttle.use) {
 						start(request, 'throttle');
-						const entry = global.server.throttle[request.ip]?.[endpoint]?.[method];
-						if (entry) {
-							if (entry.exp < Date.now()) {
-								entry.exp = Date.now() + throttle.ttl * 1000;
-								entry.hits = 0;
-							}
-							entry.hits += 1;
-							if (entry.hits > throttle.limit) {
-								debug(`throttle limit on route ${endpoint}`);
-								const status = 429;
-								stop(request, 'throttle');
-								return createResponse({ status, message: 'too many requests' }, status, request, {
-									'retry-after': `${Math.ceil((entry.exp - Date.now()) / 1000)}`,
-								});
+						if (config.throttleTtl === throttle.ttl && config.throttleLimit === throttle.limit) {
+							const glob = global.server.throttle[request.ip]?.['']?.[''];
```

Comment:
maybe call it `globalThrottle` as glob is weird and global is taken?

Reply:
Yeah global is taken because environment defines it. I need to rename entry too...